### ISSUE TITLE
Error on setMenu

### ISSUE THREAD

**Issue by maxisam** [onehungrymind/electrogram#4]
![image](https://cloud.githubusercontent.com/assets/456807/16022879/b131020a-317f-11e6-9f1e-d9f0e01b04ad.png)

I got this when I run main. Any suggestion ?

it also shows electron could not be found. no hard resets for you in console.

however, it seems like it can still run after that error message

**Comment by michaeljota** [comment]
I think it's because the entire menu template have been made just for OS X. 

``` javascript
if (process.platform == 'darwin') {
...
} //No else here. 
```

I got the same error. Did you manage to solve it? Was this?

---

Edit: It wasn't. You have to check if this if running in other OS rather than OS X, before the dock menu is being armed. You left those parts out, and I'll run. Be sure to build first, though.

**Comment by maxisam** [comment]
I was on Win10 not OS X

**Comment by michaeljota** [comment]
Yes. Me too. But the developer did not catch any plataform related issue, so when you try to build, electron tries to build the dock menu, but that's only avaliable to mac.

Change this

``` javascript
  if (!dockMenu && process.platform === 'darwin') setDock(); //~line 40
...
  if (process.platform == 'darwin'){                         //~line 62
    let dockWindowMenu = dockMenu.items
      .find(item => item.label === 'New Window');
    dockWindowMenu.enabled = isEnabled;
  }
```

I may edit something else, but I think with this you could be able to run it.
